circuits/cpp/src/aztec3/circuits/abis/private_circuit_public_inputs.hpp

```diff
@@ -76,10 +76,10 @@ template <typename NCT> class PrivateCircuitPublicInputs {
     {
         return call_context == other.call_context && args_hash == other.args_hash &&
                return_values == other.return_values && read_requests == other.read_requests &&
-               new_commitments == other.new_commitments &&
-               new_nullifiers == other.new_nullifiers && nullified_commitments == other.nullified_commitments &&
-               private_call_stack == other.private_call_stack && public_call_stack == other.public_call_stack &&
-               new_l2_to_l1_msgs == other.new_l2_to_l1_msgs && encrypted_logs_hash == other.encrypted_logs_hash &&
+               new_commitments == other.new_commitments && new_nullifiers == other.new_nullifiers &&
```

Comment:
not sure why this is being shown here

Reply:
i think there's an issue with how you merged/rebased, maybe worth reverting the merge and trying again so you don't appear as the author of this